modules/gothamadsBidAdapter.js

```diff
@@ -1,12 +1,27 @@
-import { registerBidder } from '../src/adapters/bidderFactory.js';
-import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+import {
+  registerBidder
+} from '../src/adapters/bidderFactory.js';
+import {
+  BANNER,
+  NATIVE,
+  VIDEO
+} from '../src/mediaTypes.js';
 import * as utils from '../src/utils.js';
-import {config} from '../src/config.js';
+import {
+  config
+} from '../src/config.js';
```

Comment:
This is personal preference so you don't have to change it if you disagree, but I think previously imports were more readable (when import was in single line, not in multiple lines).